hazelcast/src/main/java/com/hazelcast/client/config/ClientConfigRecognizer.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Shouldn't be the community license? 🤔

Reply:
For OS we kept apache, community is for integration modules mostly.